x-pack/package.json

```diff
@@ -44,12 +44,13 @@
     "@mapbox/mapbox-gl-draw": "^1.2.0",
     "@mapbox/mapbox-gl-rtl-text": "^0.2.3",
     "@scant/router": "^0.1.0",
-    "@storybook/addon-actions": "^6.0.16",
-    "@storybook/addon-essentials": "^6.0.16",
-    "@storybook/addon-knobs": "^6.0.16",
-    "@storybook/addon-storyshots": "^6.0.16",
-    "@storybook/react": "^6.0.16",
-    "@storybook/theming": "^6.0.16",
+    "@storybook/addon-a11y": "^6.0.26",
```

Comment:
Let's just put these in the root package.json. It's really weird to expect these in the root node_modules when they're listed in the x-pack/package.json file. Sorry for the runaround!